index.html

```diff
@@ -74,6 +75,7 @@
                 </div>
             </div>
         </div>
+        <script src="http://labratrevenge.com/d3-tip/javascripts/d3.tip.v0.6.3.js"></script>
```

Comment:
Remove this line. d3.tip is included below. This one causes a JavaScript error because d3 hasn't been included yet.